lib/mayaUsd/render/vp2ShaderFragments/shaderFragments.cpp

```diff
@@ -345,6 +350,15 @@ MStatus HdVP2ShaderFragments::registerFragments()
     }
 
     {
+        // This is a temporary fix for Maya 2022 that is quite fragile and will need to be revisited
+        // in the near future. It will not handle the custom color space some large scale clients
+        // use. It will also not handle any custom OCIO config that could change the color
+        // interpolation algorithm.
```

Comment:
Below is my suggestion for the comment to add:

Note: Because the supported color spaces are hard-coded (i.e names and color transformation implementation) it can only support few color spaces from the Maya default configs and it will definitively fail on any custom configs. So, this is a temporary fix for Maya that is quite fragile and will need to be revisited in the near future.